test/fixtures/dialects/athena/create_external_table_orc_file_type.sql

```diff
@@ -0,0 +1,13 @@
+CREATE EXTERNAL TABLE my_table(
```

Comment:
This file doesn't appear to use the new `ORCFILE` keyword. Wasn't that the intent of adding this test file, to ensure that the new keyword parses correctly?

Reply:
This PR has become confusing because there was no associated issue. In your PR, you mention that the linter is failing with a specific error, but _you didn't provide an example of a file that reproduced the failure_`. That's what we need here. I can see that this file would use ORC if it were executed in Athena, but that's not relevant. We need a test case that actually uses the `ORCFILE` keyword. You must've had a file like that which was failing, right? **That** file can be the test case. (Preferably a simplified/minimized version of it.)

Reply:
Ahhh I know get whats happening. It is acutally not the sql query it is the implementation. The problem is `STORED AS INPUTFORMAT`. Once it is set it will check grammar and the keyword segements. It is failing because Orc is mentioned in grammar but not as Keyword.

So it is actually not about orc but the `INPUTFORMAT` key word.

Btw: The query is kind of the one that was failing - but now it's clear why.


I will remove the test and create one that checks for `STORED AS INPUTFORMAT`. The formats itself should be tested as well I guess but that should be another PR.